client/src/locales/translations/en_GB.json

```diff
@@ -267,7 +267,7 @@
       "myActivity": "My Activity",
       "userActivity": "User Activity",
       "wantHelp": "I want to help",
-      "needHelp": "I need help",
+      "needHelp": "I need help ",
```

Comment:
A trailing space here but I think that's fine

Reply:
I saw this trailing space in Eng-US.

Reply:
Then maybe it's supposed to be there.